frontend/e2e-tests/MoEmployeeCreate.js

```diff
@@ -122,14 +124,13 @@ test('Workflow: create employee', async t => {
     .click(dialog.find('.btn-address .btn-outline-success'))
 
     .click(addressTypeSelect)
-    .click(addressTypeOption.nth(1))
+    .click(addressTypeOption.nth(6))
```

Comment:
Can we pick it by value, i.e. `withText` or similar?

Reply:
Sure!